river: Add various string functions to stdlib (#4388)
grafana/agent: docs/sources/flow/reference/stdlib/lower.md

---
aliases:
- ../../configuration-language/standard-library/lower/
title: lower
---

# lower

> `lower` converts all cased letters in the given string to lowercase.

Reviewer: ```suggestion
`lower` converts all uppercase letters in a string to lowercase.
```
